GlyssenEngine/GlyssenEngine.csproj

```diff
@@ -39,19 +45,32 @@
     <None Remove="Resources\RelatedCharacters.xml" />
     <None Remove="Resources\SampleMRK.xml" />
     <None Remove="Resources\Silent.wav" />
+    <None Include="..\LICENSE">
```

Comment:
If you use `<PackageLicenseExpression>MIT</PackageLicenseExpression>` above you don't have to include this file.